tests/cypress/integration/actions_tasks/case_119_change_status_task.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2021-2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Task status change.', () => {
+    const caseId = '119';
+    const labelName = `Case ${caseId}`;
+    const taskName = `New annotation task for ${labelName}`;
+    const attrName = `Attr for ${labelName}`;
+    const textDefaultValue = 'Some default value for type Text';
+    const imagesCount = 15;
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const advancedConfigurationParams = {
+        multiJobs: true,
+        segmentSize: 5,
+    };
```

Comment:
Consider code from ``cvat/tests/cypress/integration/masks/masks_basics.js`` to create a task, it works much faster, because it is being done in headless mode

Reply:
`cy.headlessCreateTask({
            labels: [{ name: labelName, attributes: [], type: 'any' }],
            name: taskName,
            // TODO: add support for multi jobs
            // segments: [{
            //     jobs: []
            // }],
            project_id: null,
            source_storage: { location: 'local' },
            target_storage: { location: 'local' },
        }, {
            server_files: serverFiles,
            image_quality: 70,
            use_zip_chunks: true,
            use_cache: true,
            sorting_method: 'lexicographical',
        }).then((response) => {
            console.log(response);
            taskId = response.taskID;
        }).then(() => {
            cy.visit(`/tasks/${taskId}`);
        });`

Using the option you suggested, the method creates only one job attached.

Can you highlight which object to send to create a Task with multijob ?

Reply:
Need to specify ``segment_size: 1``` after ``sorting_method`` for example.